draft-ietf-cose-merkle-tree-proofs.md

```diff
@@ -472,12 +498,15 @@ See the security considerations section of:
 
 ## Choice of Signature Algorithms
 
-A security analysis SHOULD be performed to ensure that the
+A security analysis MUST be performed to ensure that the
 digitial signature algorithm `alg` is the appropriate strength to secure receipts.
 
+It is recommended to select signature algorithms that share cryptographic components with the verifiable data structure used, for example:
+Both RFC9162_SHA256 and ES256 depend on the sha-256 hash function.
+
 ## Validity Period
 
-In some cases, receipts SHOULD have strict validity periods, for example,
+In some cases, receipts might require strict validity periods, for example,
```

Comment:
"Might require" is unclear; I believe this refers to the inclusion of a validity period in the information authenticated by the receipt, not an abstract requirement. May go back to "may include" ?